assert/assertions.go

```diff
@@ -263,6 +272,23 @@ func CallerInfo() []string {
 			isTest(name, "Example") {
 			break
 		}
+
+		if !more {
```

Comment:
```suggestion
		if more {
         continue
    }
```

This would be simpler and more readable, no?